static-build/qemu-virtiofs/Dockerfile

```diff
@@ -8,11 +8,13 @@ ARG PREFIX
 
 WORKDIR /root/qemu-virtiofs
 RUN apt-get update && apt-get upgrade -y
-RUN apt-get install -y \
+RUN apt-get --no-install-recommends install -y \
+		apt-utils \
 	    autoconf \
 	    automake \
 	    bc \
 	    bison \
+		ca-certificates \
```

Comment:
I think that dependencies should be aligned

Reply:
done